snet/cli/contract.py

```diff
@@ -22,6 +25,6 @@ def process_receipt(self, receipt):
 
         contract_events = map(lambda e: e["name"], filter(lambda e: e["type"] == "event", self.abi))
         for contract_event in contract_events:
-            events.extend(getattr(self.contract.events, contract_event)().process_receipt(receipt))
+            events.extend(getattr(self.contract.events, contract_event)().process_receipt(receipt, errors=DISCARD))
```

Comment:
Here we ignoring UserWarnings from Web3 more about [here](https://web3py.readthedocs.io/en/stable/web3.contract.html#web3.contract.ContractEvents)